src/pamiq_core/threads/thread_control.py

```diff
@@ -1,4 +1,9 @@
+import logging
 import threading
+from concurrent.futures import Future, ThreadPoolExecutor
+
+from pamiq_core.threads.thread_types import ThreadTypes
+from pamiq_core.utils import get_class_module_path
```

Comment:
ここは直接モジュールファイルからimportするべきです。 `utils`モジュールの名前空間はなんでもアリになりやすく、一体どこから何を使用しているのか分かりにくくなってしまう問題が発生します。
```suggestion
from pamiq_core.utils.reflection import get_class_module_path
```

Reply:
なるほど、承知しました！